devices/IFrameTransformToIWear/src/IFrameTransformToIWear.cpp

```diff
@@ -0,0 +1,518 @@
+/*
+ * Copyright (C) 2021 Istituto Italiano di Tecnologia (IIT)
+ * All rights reserved.
+ *
+ * This software may be modified and distributed under the terms of the
+ * GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#include "IFrameTransformToIWear.h"
+
+#include <yarp/dev/IFrameTransform.h>
+#include <yarp/os/LogStream.h>
+#include <yarp/sig/Vector.h>
+
+#include <map>
+#include <mutex>
+#include <string>
+#include <vector>
+
+const std::string DeviceName = "IFrameTransformToIWear";
+const std::string LogPrefix = DeviceName + " :";
+
+using namespace wearable;
+using namespace wearable::sensor;
+using namespace wearable::devices;
+
+struct ParsedOptions
+{
+    wearable::WearableName wearableName;
+    wearable::sensor::SensorType wearableSensorType;
+    std::string rootFrameID;
+
+    std::vector<std::string> frameIDs;
+};
+
+// Class that provides each frame information, and utilites
+// to map its data to containers compatible with IWear
+class IFrameTransformHandler
```

Comment:
I don't see the initialization of the yarp network. If this is necessary, you also to link against `YARP::YARP_init`.

Reply:
I am a bit confused about where this initialization should be done. I think it is possibly missing also on other devices (e.g. [`IAnalogSensorToIWear`](https://github.com/robotology/wearables/tree/master/devices/IAnalogSensorToIWear) since I don't see it.

Reply:
As I often repeat, do not trust too much what was done in the past. Mistakes are always behind the corner.

In short, the initialization of the network should be done at least once per process. I suspect that `yarprobotinterface` already takes care of that, but to be safe I'd add that logic to all devices (which are plugins, i.e. shared libraries loaded during runtime with dlopen). cc @Yeshasvitvs

Reply:
> In short, the initialization of the network should be done at least once per process. I suspect that `yarprobotinterface`

Yes, that is why I though it was not necessary, I see your point though, I think we can add the check in the `open()` method, what do you think?

> As I often repeat, do not trust too much what was done in the past. Mistakes are always behind the corner.

Sure, after merging this PR I will eventually updated the other devices where this check is missing.

Reply:
I have updated also the other devices in https://github.com/robotology/wearables/pull/126/commits/8a0eeba6066fb458027036980841aad01075a0cc